fabfile.py

```diff
@@ -3483,52 +3483,61 @@ def fusion_perf_test(cluster, n=3, keepRunning=False, instance_type='r3.2xlarge'
-    _status('SolrCloud cluster provisioned ... deploying the perf config directory to ZK as name: perf')
-    deploy_config(cluster,'perf','perf')
-    deploy_config(cluster,'perf','perf_js')
-    _status('Starting Fusion services across cluster ...')
-    fusion_start(cluster,api=n,connectors=1,ui=n,yjp_path=yjp_path_fusion,apiJavaMem=apiJavaMem)
-    hosts = _lookup_hosts(cluster)
+    print "Starting hacked up solr test..."
+
+    # yjp_path_solr = None
+    # yjp_path_fusion = None
+    # if yjpPath is not None:
+    #     if bool(yjpSolr) is True:
+    #         yjp_path_solr = yjpPath
+    #     if bool(yjpFusion) is True:
+    #         yjp_path_fusion= yjpPath
+    #
+    # solrJavaMemOpts = None
+    # apiJavaMem = None
+    # if instance_type == 'r3.xlarge':
+    #     solrJavaMemOpts = '-Xms6g -Xmx6g'
+    #     apiJavaMem = '3g'
+    # elif instance_type == 'r3.2xlarge':
+    #     solrJavaMemOpts = '-Xms8g -Xmx8g'
+    #     apiJavaMem = '4g'
+    #
+    # hvmAmiId = _env(cluster, 'AWS_HVM_AMI_ID')
+    # _info('Starting Fusion cluster using HVM-based AMI: '+hvmAmiId)
+    #
+    # new_solrcloud(cluster,
+    #               n=n,
+    #               zkn=min(3,n),
+    #               ami=hvmAmiId,
+    #               instance_type=instance_type,
+    #               placement_group=placement_group,
+    #               yjp_path=yjp_path_solr,
+    #               auto_confirm=True,
+    #               solrJavaMemOpts=solrJavaMemOpts,
+    #               purpose='Fusion Performance Testing',customTags='{"CostCenter":"eng"}')
+    # _status('SolrCloud cluster provisioned ... deploying the perf config directory to ZK as name: perf')
+    # deploy_config(cluster,'perf','perf')
+    # deploy_config(cluster,'perf','perf_js')
+    # _status('Starting Fusion services across cluster ...')
+    # fusion_start(cluster,api=n,connectors=1,ui=n,yjp_path=yjp_path_fusion,apiJavaMem=apiJavaMem)
+    #hosts = _lookup_hosts(cluster)
+
+    # hardcoding the hostname of where solr can be reached
+    hosts = ["ui-staging.k8s.lucidworks.io"]
+    host_port = "80" # "8764"
 
     # make sure the proxy / UI service is up before making changes with the API
-    _wait_to_see_fusion_proxy_up(cluster, hosts[0], 60)
-    _status('Fusion proxy / UI service is up, commencing with post-startup configuration steps ...')
+    #_wait_to_see_fusion_proxy_up(cluster, hosts[0], 60)
+    #_status('Fusion proxy / UI service is up, commencing with post-startup configuration steps ...')
 
     # set the initial password for the new Fusion cluster
     fusionPasswd = '{"password":"password123"}'
-    postToApiUrl = "http://%s:8764/api" % hosts[0]
+    postToApiUrl = "http://%s:%s/api" % (hosts[0], host_port)
     req = urllib2.Request(postToApiUrl)
     req.add_header('Content-Type', 'application/json')
+
+    print "Connecting to: "+str(postToApiUrl)
+
     try:
         urllib2.urlopen(req, fusionPasswd)
     except urllib2.HTTPError as e:
```

Comment:
Looks like it actually sets the password but does return the 409.  Commenting this part out and continuing on.